tests/mgbench/benchmark.py

```diff
@@ -198,87 +242,128 @@ def filter_benchmarks(generators, patterns):
-                      args.single_threaded_runtime_sec,
-                      "seconds of single-threaded runtime...")
-                # First run to prime the query caches.
-                memgraph.start_benchmark()
-                client.execute(queries=get_queries(func, 1), num_workers=1)
-                # Get a sense of the runtime.
-                count = 1
-                while True:
-                    ret = client.execute(queries=get_queries(func, count),
-                                         num_workers=1)
-                    duration = ret[0]["duration"]
-                    should_execute = int(args.single_threaded_runtime_sec /
-                                         (duration / count))
-                    print("executed_queries={}, total_duration={}, "
-                          "query_duration={}, estimated_count={}".format(
-                              count, duration, duration / count,
-                              should_execute))
-                    # We don't have to execute the next iteration when
-                    # `should_execute` becomes the same order of magnitude as
-                    # `count * 10`.
-                    if should_execute / (count * 10) < 10:
-                        count = should_execute
-                        break
-                    else:
-                        count = count * 10
-                memgraph.stop()
-                config.set_value(*config_key, value={
-                    "count": count,
-                    "duration": args.single_threaded_runtime_sec})
-            else:
-                print("Using cached query count of", cached_count["count"],
-                      "queries for", cached_count["duration"],
-                      "seconds of single-threaded runtime.")
-                count = int(cached_count["count"] *
-                            args.single_threaded_runtime_sec /
-                            cached_count["duration"])
+    for with_fine_grained_authorization in [False, True]:
+        if with_fine_grained_authorization:
+            memgraph.start_preparation()
+            client.execute(file_path=dataset.get_file(), num_workers=args.num_workers_for_import)
+            client.execute(
+                queries=[
+                    ("CREATE USER user IDENTIFIED BY 'test';", {}),
+                    ("GRANT ALL PRIVILEGES TO user;", {}),
+                    ("GRANT CREATE_DELETE ON EDGE_TYPES * TO user;", {}),
+                    ("GRANT CREATE_DELETE ON LABELS * TO user;", {}),
+                ]
+            )
+            client = runners.Client(
+                args.client_binary,
+                args.temporary_directory,
+                args.bolt_port,
+                username="user",
+                password="test",
+            )
+            memgraph.stop()
+
+        test_type = (
+            WITH_FINE_GRAINED_AUTHORIZATION if with_fine_grained_authorization else WITHOUT_FINE_GRAINED_AUTHORIZATION
+        )
```

Comment:
what does this do in the end? we are not logging properly the results in the end if we can't distinct what was with authorization and what was not